bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -155,16 +155,28 @@ metadata:
     createdAt: "2020-09-08T12:21:00Z"
     description: OADP (OpenShift API for Data Protection) operator sets up and installs
       Data Protection Applications on the OpenShift platform.
-    features.operators.openshift.io/token-auth-aws: "true"
     olm.skipRange: '>=0.0.0 <99.0.0'
     operatorframework.io/suggested-namespace: openshift-adp
-    operators.openshift.io/infrastructure-features: '["Disconnected"]'
     operators.openshift.io/valid-subscription: '["OpenShift Container Platform", "OpenShift
       Platform Plus"]'
     operators.operatorframework.io/builder: operator-sdk-v1.23.0
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/openshift/oadp-operator
     support: Red Hat
+    # DEPRECATED but still functional starting with OCP v4.14
+    operators.openshift.io/infrastructure-features: '["Disconnected"]'
+    # New infrastructure annotation format
+    # MUST provide either the string value (not boolean) "true" or "false" for each
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "true"
+    features.operators.openshift.io/proxy-aware: "true"
+    features.operators.openshift.io/cnf: "false"
+    features.operators.openshift.io/cni: "false"
+    features.operators.openshift.io/csi: "true"
```

Comment:
Does this mean CSI Driver? is there a document that says what each of these means?

Reply:
@shawn-hurley , linked at the [top](https://docs.engineering.redhat.com/display/CFC/Best_Practices#Best_Practices-(New)RequiredInfrastructureAnnotations) is what I got.  That... and a breadth of Jira Issues you could go chase down. :sweat_smile:

Reply:
> Whether the operator provides a Container Storage Interface (CNI) Kubernetes plugin.

I don't believe that we do provide a CSI plugin I think this should stay false.